js/components/Editor.js

```diff
@@ -124,12 +124,13 @@ const Editor = React.createClass({
 
   render() {
     return (
-      dom.div(
-        { className: "editor" },
-        dom.textarea({
-          ref: "editor",
-          defaultValue: "..."
-        })
+      dom.div({ className: "editor-container" },
```

Comment:
sadly that doesn't work due to a chrome flex box bug with height and width's with percentages.